### PR TITLE
Fixed Number Input Field Issue.

### PR DESCRIPTION
**Pull request by S-Abdul-Rahman**
Fixed Number Input Field Issue #7414

### CODE REVIEW

File: frontend/src/Editor/Components/NumberInput.jsx [hunk 1]
```diff
@@ -49,9 +49,10 @@ export const NumberInput = function NumberInput({
     fireEvent('onChange');
   };
   const handleBlur = (e) => {
-    setValue(Number(parseFloat(e.target.value).toFixed(properties.decimalPlaces)));
+    setValue(Number(parseFloat(e.target.value ? e.target.value : 0).toFixed(properties.decimalPlaces)));
```
Comment (Ckbhatia) [inline frontend/src/Editor/Components/NumberInput.jsx:52]: What was the issue with any number?

**Review by kavinvenkatachalam — CHANGES_REQUESTED** [review]
@S-Abdul-Rahman Can you resolve the conflicts?

**Comment by S-Abdul-Rahman** [comment]
> @S-Abdul-Rahman Can you resolve the conflicts?

@kavinvenkatachalam yes, resolved the conflicts.

**Review by kavinvenkatachalam — APPROVED** [review]

**Comment by Nandinisaha13** [comment]
@S-Abdul-Rahman Can you please check why can't we show 0 value from number input, attaching a video for reference. 

Uploading Screen Recording 2023-11-30 at 12.02.34 PM.mov…

**Pull request merged**